src/screens/QuestionScreen.tsx

```diff
@@ -0,0 +1,272 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  ScrollView,
+  Alert,
+} from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { useTaskContext } from '../context/TaskContext';
+import { TaskScore } from '../types';
+
+const QuestionScreen: React.FC = () => {
+  const navigation = useNavigation();
+  const { questions, tasks, currentScores, updateTaskScore, addAnsweredQuestion, resetScores } = useTaskContext();
+  const [currentQuestionIndex, setCurrentQuestionIndex] = useState(0);
+
+  const currentQuestion = questions[currentQuestionIndex];
+
+  const handleAnswer = (answer: boolean) => {
+    if (!currentQuestion) return;
```

Comment:
is this just for safety? if there's no current question how would we call handleAnswer?